pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -66,6 +66,8 @@ def test_get_loc(self):
         for method, loc in [('pad', 1), ('backfill', 2), ('nearest', 1)]:
             assert idx.get_loc('1 day 1 hour', method) == loc
 
+        assert idx.get_loc(idx[1].to_timedelta64()) == 1
```

Comment:
Could you add a comment pointing to the issue that caused you to write this assertion e.g.:

~~~
see #16909
~~~

OR

~~~
see gh-16909
~~~